### PR TITLE
Frontend_V2: Show execution time and submitted Time at My Submission row(#3337)

### PR DESCRIPTION
**Pull request by savish28**
Issue Fix: #3337
For each entry in the `My submissions` table moved `Execution time` and `Submitted at` out of collapsible div and showed it for each row

Before:
Screenshot from 2021-03-24 15-05-48

After:
Screenshot from 2021-03-24 15-03-07

### CODE REVIEW

**Comment by savish28** [comment]
@Ram81 ping. PR ready for review.

File: frontend_v2/src/app/components/challenge/challengesubmissions/challengesubmissions.component.ts [hunk 1]
```diff
@@ -151,8 +151,8 @@ export class ChallengesubmissionsComponent implements OnInit, AfterViewInit {
    */
   apiCall: any;
 
-  columnsToDisplay = ['s_no', 'status', 'submission_result_file', 'stdout_file', 'stderr_file'];
-  columnsHeadings = ['S.No.', 'Status', 'Result File', 'Stdout File', 'Stderr File', ];
+  columnsToDisplay = ['s_no', 'status', 'execution_time', 'submitted_at', 'submission_result_file', 'stdout_file', 'stderr_file'];
+  columnsHeadings = ['S.No.', 'Status', 'Execution Time', 'Submitted At', 'Result File', 'Stdout File', 'Stderr File'];
```
Comment (Ram81) [inline frontend_v2/src/app/components/challenge/challengesubmissions/challengesubmissions.component.ts:155]: @savish28 let's move `Submitted At` to the end of the table.

**Review by Ram81 — CHANGES_REQUESTED** [review]

File: frontend_v2/src/app/components/challenge/challengesubmissions/challengesubmissions.component.ts (hunk 1 above)
Comment (savish28) [reply frontend_v2/src/app/components/challenge/challengesubmissions/challengesubmissions.component.ts]: @Ram81 Done.

**Comment by Ram81** [comment]
@savish28 can you share updated screenshot of the page.

**Comment by savish28** [comment]
> @savish28 can you share updated screenshot of the page.

Sure, Here it is.
Screenshot from 2021-03-24 20-06-08

**Comment by Ram81** [comment]
@savish26 3rd row shows `None sec` can we please fix it. If execution time is none we should show `N/A`

**Comment by savish28** [comment]
> @savish26 3rd row shows `None sec` can we please fix it. If execution time is none we should show `N/A`

Yup, Now it looks like, (Attaching the screenshot).

Screenshot from 2021-03-25 20-35-39

**Comment by Ram81** [comment]
@savish28 the execution time for failed submissions should also be there. Can you please confirm why the value is being populated as `None`, as we calculate execution time using `submitted_at` and `completed_at` date fields of a submission it should not be `None`.

**Comment by savish28** [comment]
> @savish28 the execution time for failed submissions should also be there. Can you please confirm why the value is being populated as `None`, as we calculate execution time using `submitted_at` and `completed_at` date fields of a submission it should not be `None`.

Yes, I was also looking for the same thing. I am unsure of how the thing appeared on the first place.  But to re-create the problem  I had set the value of `submitted_at` and `completed_at` to empty from admin.

So as you suggested it would be good to safeguard this by showing N/A instead of None sec.

**Comment by Ram81** [comment]
>@Ram81, I was also looking for the same thing. I am unsure of how the thing appeared on the first place. But to re-create the problem I had set the value of submitted_at and completed_at to empty from admin.

This case never happens. Isn't the data in the screenshot from the seed script? or did you make new submissions?

**Comment by savish28** [comment]
@Ram81 I made a new submission. The seed data doesn't contain any None case. When in seed data I changed the start time to empty it then showed None sec.

**Review by Ram81 — APPROVED** [review]
LGTM 👍

**Pull request merged**